core/lib/spree/core/importer/product.rb

```diff
@@ -7,8 +7,8 @@ class Product
         def initialize(product, product_params, options = {})
           @product = product || Spree::Product.new(product_params)
 
-          @product_attrs = product_params
-          @variants_attrs = options[:variants_attrs] || []
+          @product_attrs = product_params.to_h
+          @variants_attrs = (options[:variants_attrs] || []).map(&:to_h)
```

Comment:
What about Rails 5 made this necessary?  It feels like this should be done at the controller level to me.